src/steamship/cli/cli.py

```diff
@@ -758,6 +759,43 @@ def support_info():
     click.secho(f"In virtual env: {sys.prefix != sys.base_prefix}")
 
 
+@click.command()
+@click.option(
+    "--all",
+    is_flag=True,
+    default=False,
+    help="Delete all local development instances without prompting",
+)
+def delete_local_dev_instances(all: Optional[bool] = False):
```

Comment:
Nit: naming differences between "delete" here vs "clean" in argument name

Reply:
👍